### ISSUE TITLE
Label for Task Template Tile

### ISSUE THREAD

**Issue by use-to** [helpwave/web#318]
We need a label in the upper right corner.
<img width="1440" alt="Tasks - overview with details pane with task modal" src="https://user-images.githubusercontent.com/13922835/235243382-135b27a6-b63d-4a52-885a-4781a765db84.png">

**Comment by shivadharmi** [comment]
@use-to Label you meant here is the one that is on top of the template tile with `ward` or `person`. Is that correct?

**Comment by MaxSchaefer** [comment]
> @use-to Label you meant here is the one that is on top of the template tile with `ward` or `person`. Is that correct?

Correct @shivadharmi, I've updated the description.

**Comment by shivadharmi** [comment]
@MaxSchaefer Should I render the edit icon conditionally based on whether is function is defined or not? Or Should I pass another variable `showEdit` to render it.

**Comment by MaxSchaefer** [comment]
> @MaxSchaefer Should I render the edit icon conditionally based on whether is function is defined or not? Or Should I pass another variable `showEdit` to render it.

Hey @shivadharmi,
the edit button gets already introduced at https://github.com/helpwave/web/commit/7ddd6ff446095861e92eb51696091b34f861a78c. The label can be a pill-label as in the screenshot and should be optional. The text inside the label should be configurable via a prop. Optional means, no text gets passed.

**Comment by MaxSchaefer** [comment]
@shivadharmi are you working on this?

**Comment by shivadharmi** [comment]
Yes, but I got stuck on the label colors. I need help on that part.
I making a draft PR.
